include/TS3.h

```diff
@@ -16,14 +16,14 @@ class TS3 : public TGRSIDetector {
 	public:
 
 		enum ES3Bits {
-			kPixelsSet 			= BIT(0),
-			kMultHit     	 	= BIT(1),
-			kBit2        		= BIT(2),
-			kBit3         		= BIT(3),
-			kBit4         		= BIT(4),
-			kBit5  				= BIT(5),
-			kBit6   				= BIT(6),
-			kBit7   				= BIT(7)
+			kPixelsSet	= BIT(0),
+			kMultHit	= BIT(1),//Should this be moved to a static?
+			kPreSector	= BIT(2),//Should this be moved to a static?
```

Comment:
Ah fair enough. Yes, if it's a "should we do this in S3 always?" type flag, then it should be made static.